src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
```

Comment:
Using @author tag is not recommended as it can be extracted from version control. (Severity: low)